macros/contexts/contextBoolean.pl

```diff
@@ -0,0 +1,160 @@
+## contextBoolean.pl
+
+sub _contextBoolean_init { context::Boolean::Init() }
+
+package context::Boolean;
+
+sub Init {
+	my $context = $main::context{Boolean} = Parser::Context->getCopy("Numeric");
+	$context->{name} = "Boolean";
+	$context->{value}{Formula} = "context::Boolean::Formula";
+
+	## Disable unwanted operations and functions
+	$context->functions->disable('All');
+	#    $context->parens->undefine('<');
+	$context->operators->undefine('-', '/', ' /', '/ ', '//', '^', '**', 'u+', '!');
+	$context->operators->redefine('><');
+
+	## Re-enable necessary operations
+	$context->operators->set(
+		'+'   => { class => "Parser::BOP::or",  string => ' or ',  TeX => '\vee ',   perl => '||' },
+		'or'  => { class => "Parser::BOP::or",  string => ' or ',  TeX => '\vee ',   perl => '||' },
```

Comment:
`or` could just be `{ alias => '+' }`. Or make `+` be the alias for `or`, whichever seems better.

Same with the `and` operators. And I could be wrong, but I don't think you need the ` *` and `* ` variants. I think that you only see those in other places because Davide had a plan to make them behave slightly differently than `*` in some situations. Without having ` *`, I think that an actual ` *` would collapse the space and just be parsed as `*`.

Reply:
I like this suggestion 👍